src/components/TextField/TextField.tsx

```diff
@@ -158,10 +158,15 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
         if (inputRef.current) {
           inputRef.current.focus();
         }
-        clearValue();
+        inputRef.current.value = "";
+        const syntheticChangeEvent = {
+          type: "change",
+          target: inputRef.current
```

Comment:
btw, is this what Lior on ask-vibe meant? It sounded like he wanted to send a name to the `TextField` component and receive it back from the `onChange`